vue-frontend/src/components/partials/NewHeader.vue

```diff
@@ -164,17 +164,14 @@
           >
         </li>
       </ul>
-      <div
-        v-if="showMenu"
-        class="tw-fixed tw-bottom-0 tw-left-0 tw-w-full tw-h-auto tw-bg-white tw-p-4 tw-shadow-[0_25px_50px_-12px_rgb(0,0,0)]"
-      >
+      <div v-if="showMenu" class="tw-w-full tw-h-auto tw-bg-white tw-p-4">
         <div className="tw-grid tw-p-3 tw-pb-5">
           <a
             href="/contact"
-            className="tw-relative tw-justify-self-center tw-rounded-[8px] tw-border-[1px] tw-border-solid tw-border-transparent tw-bg-gradient-to-r tw-from-[#f2709c] tw-to-[#ff9472] hover:tw-shadow-[inset_2px_1000px_1px_#fff] tw-font-bold tw-text-white "
+            className="tw-relative tw-justify-self-center tw-rounded-full tw-border-[1px] tw-border-solid tw-border-transparent tw-bg-gradient-to-r tw-from-[#f2709c] tw-to-[#ff9472] hover:tw-shadow-[inset_2px_1000px_1px_#fff] tw-font-bold tw-text-white "
           >
             <div
-              className="tw-py-[0.8rem] tw-px-[2.5rem] tw-text-lg tw-gradient-text tw-inline-block"
+              className="tw-py-[0.8rem] tw-px-[2.5rem] tw-text-lg gradient-text tw-inline-block"
```

Comment:
We are using `tw- ` prefix for tailwind classes, otherwise classes will not be applied

Reply:
i have create class in App.vue for this gradient text.